src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -72,7 +82,8 @@ export default {
             parentKeyString: this.openmct.objects.makeKeyString(this.domainObject.identifier),
             conditionCollection: [],
             conditions: [],
-            currentConditionIdentifier: this.currentConditionIdentifier || {}
+            currentConditionIdentifier: this.currentConditionIdentifier || {},
+            moveIndex: null
```

Comment:
You don't need this `moveIndex` property. See comment above about not using the `set-move-index` event.

Reply:
N/A - leave as is for now.